l10n_be_cooperator_national_number/readme/USAGE.rst

```diff
@@ -0,0 +1,4 @@
+On the company, two new toggles 'Display National Number' and 'Require National
+Number' are added. In order to expose the functionality of this module, you must
+enable these toggles for every company that wishes to use it. 'Display' shows
+the field and allows it to be filled in. 'Require' makes the field mandatory.
```

Comment:
Shouldn't we be able to display the field and use it without making it mandatory ? If not what's the point in having two toggles ?

Reply:
Ok, it's just that the sentence "In order to expose the functionality of this module, you must enable these toggles for every company that wishes to use it.", seems to mean that you **have to** enable the two toggles for the functionality to work, while actually you can only toggle "display" and the functionality will work.

Reply:
Ah, maybe let's write 'one or both of these toggles'?